Python/Product/Django/TemplateParsing/TemplateArtifactCollection.cs

```diff
@@ -29,11 +29,14 @@ public TemplateArtifactCollection()
             LeftSeparator = RightSeparator = "";
         }
 
-        protected override ISensitiveFragmentSeparatorsInfo SeparatorInfo {
-            get {
-                return this;
-            }
-        }
+        // HACK: Does not build properly on Dev12

```

Comment:
is this interesting for this case?  Also, there is another bit of dead code (analysis dictionary) we might want to remove in both 2.2 and 2.1